smslibrary/src/test/java/com/eis/smslibrary/SMSHandlerTest.java

```diff
@@ -0,0 +1,41 @@
+package com.eis.smslibrary;
+
+import android.telephony.SmsManager;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
```

Comment:
Consider adding the author also in test classes, with a simple javadoc comment to the class.
In this case it could be like this:
```suggestion
/**
 * Test class for SMSHandler.
 *
 * @author Marco Cognolato
 */
```

Reply:
WOOO I didn't know about this, when you suggest code changes (obv using the proper format) I can directly commit those with the press of a button. Beautiful ❤️